### ISSUE TITLE
spyware

### ISSUE THREAD

**Issue by Amazakay** [MarioCrane/LeaueLobby#5]
it says that have spyware

**Comment by MarioCrane** [comment]
Are you sure open source software has spyware?

**Comment by Shuunen** [comment]
as @MarioCrane say it's hard to hide spyware in an open source software

I just review the source code, nothing malicious AFAIK

the only thing is I can't review `libcurl_a.lib`

i'm not familiar with C, is including third party lib is mandatory ? I used to work with C# where you could import things without having to embed stuff like this, also on JS/Node side there is a dependency management system that allows you to avoid bundling third party lib with your source code

**Comment by MarioCrane** [comment]
Thanks. libcurl_a.lib is not mandatory. I just compiled a static library. https://github.com/curl/curl

**Comment by Shuunen** [comment]
![image](https://user-images.githubusercontent.com/439158/118809187-93e2d680-b8aa-11eb-853d-c1ae5f7c4a05.png)

https://www.virustotal.com/gui/file/18ec6146b371d55a0526abfd03b434c1f8fa0a0569bd6d5fa20ef40ace1115da/detection

that's weird 🤔

**Comment by Shuunen** [comment]
Anyway I tried to compile your sources myself and lost more time doing this than doing my own version in Node Js 😋 

If anyone is interested, only 60 lines of javascript, no dependencies, nothing to compile : https://github.com/Shuunen/snippets/blob/master/one-file/lol-practice-5v5.js

Download the file & run `node lol-practice-5v5.js "D:\Games\Riot Games\League of Legends\lockfile"`

You can also customise the game name : `node lol-practice-5v5.js "D:\Games\Riot Games\League of Legends\lockfile" "My super name"`

**Comment by sh4rkd** [comment]
> node lol-practice-5v5.js "D:\Games\Riot Games\League of Legends\lockfile
![image](https://user-images.githubusercontent.com/79306685/127968333-15944867-6ef7-4a64-81cf-707069b362fc.png)

**Comment by Shuunen** [comment]
@sh4rkd which version of node are you using? Try to rename the file extension to `.mjs`

**Comment by sh4rkd** [comment]
> .mjs

thx :)

**Comment by raimo7** [comment]
@sh4rkd, @Shuunen, @MarioCrane  does this still work? when I create the custom game, invite my friends and run the script or use the tool, it changes it to Practice Tool game, but kicks all the other people out of the lobby.

**Comment by sh4rkd** [comment]
> @sh4rkd, @Shuunen, @MarioCrane does this still work? when I create the custom game, invite my friends and run the script or use the tool, it changes it to Practice Tool game, but kicks all the other people out of the lobby.

yup work fine, in pbe open custom lobby, open invite friend (only open dont invite), run the script, now you can invite your friends and play
